server/pages/index.js

```diff
@@ -483,11 +544,7 @@ module.exports = function() {
 				} );
 
 				if ( ! section.isomorphic ) {
-					app.get(
-						pathRegex,
-						section.enableLoggedOut ? setUpRoute : setUpLoggedInRoute,
-						serverRender
-					);
+					app.get( pathRegex, setUpRoute, serverRender );
```

Comment:
This allows moving `getDefaultContext()` to `setUpRoute()`,
we're anyway checking whether to call `setUpLoggedInRoute()` or `setUpLoggedOutRoute()`
in `setUpRoute()`